integreat_cms/cms/templates/imprint/imprint_sbs.html

```diff
@@ -22,7 +22,13 @@ <h1 class="heading">
                     </a>
                     {% if perms.cms.change_imprintpage %}
                         <button name="submit_draft" class="btn btn-gray">{% trans 'Save as draft' %}</button>
-                        <button name="submit_public" class="btn">{% trans 'Publish' %}</button>
+                        <button name="submit_public" class="btn">
+                            {% if imprint_translation_form.instance.status == 'PUBLIC' %}
```

Comment:
```suggestion
                            {% if imprint_translation_form.instance.status == PUBLIC %}
```
I think to stay consistent with the other templates, we should not hardcode this value (although it is unlikely that it will change in the near future, but rather pass the `PUBLIC` variable to the template context like in the other views...
You could add e.g. `from ...constants import status` and `"PUBLIC": status.PUBLIC,` here: https://github.com/digitalfabrik/integreat-cms/blob/1b78f774ad9d766db630aaa5f975c1e3400315b9/integreat_cms/cms/views/imprint/imprint_sbs_view.py#L28